content-security-policy/generic/quote-characters.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "csp": "img-src 'none';", "expected": false, "name": "precondition: single quotes are accepted" },
+      // See https://github.com/w3c/webappsec-csp/issues/434 for background on what is tested.
+      { "csp": 'img-src "none";', "expected": true, "name": "double quotes should not be accepted" },
+      { "csp": "img-src \u0091none'", "expected": true, "name": "U+0091 + single quote should not be accepted" },
+      { "csp": "img-src 'none\u0092", "expected": true, "name": "single quote + U+0092 should not be accepted" },
+      { "csp": "img-src \u0091none\u0092", "expected": true, "name": "U+0091 + U+0092 should not be accepted" },
+      { "csp": "img-src \u2018none\u2019;", "expected": true, "name": "U+2018 + U+2019 should not be accepted" },
```

Comment:
In Chromium, these four all end up as parse failures for the directive, as it contains characters that can't exist in a directive. In this case, we punt the directive entirely. We should probably have the same behavior here that we have for `"none"` above: treat the policy as containing an `img-src` directive with no valid entries, which is equivalent to `'none'`.